### DISCUSSION TITLE
Control for front fans not working but shows RPM.

### DISCUSSION THREAD

**Discussion by jannik-lugauer** [Rem0o/FanControl.Releases#1903] [Q&A]
Hello everyone.

I upgraded my pc to new 140mm Fans and wanted to control them via Fan Control. The software detectes everything correctly except the Front two 140mm fans. Those are arctic F14 PMW PST fans. I daisy chained both of them into the fan header SYS-FAN4. 

Mainboard is the "X470 Aorus ULTRA GAMING".

Now i cant control those two Fans but i still can see the RPM from them.

![FanControl](https://github.com/Rem0o/FanControl.Releases/assets/65730267/b4f42776-5aa1-414f-971b-3706c8f54b3f)

Here is a screenshot of said Fans.

Is there any solution to this?

Thanks for the help

**Comment by Rem0o** [comment]
If that header doesn't work, you might want to try a different one if available.

**Reply by jannik-lugauer** [reply]
Tried two different headers, all of them have the same result.

**Reply by Rem0o** [reply]
Keep in mind changing header, you will also change control card in the software. The old control card you labelled won't be the good one.

**Reply by jannik-lugauer** [reply]
Yes i kept that in mind. I identified the correct control card via the speed monitors but the result is the same as with the other headers. Do you have any recommendations what to do next? Im not quite sure where to begin. Thanks in advance.
